Audio preview in the creator view editor tab no longer works

Go here: https://oppiatestserver.appspot.com/create/G8ZgSj1O4rjE?collection_id=giIwhTU3up5v#/gui/Introduction and click to start the audio playback. Nothing happens.

In contrast on the production site, here: https://www.oppia.org/create/umPkwp0L1M0-?collection_id=4UgTQUc1tala#/gui/Introduction it does work.

Audio playback should work in both cases.

@seanlip:
![image](https://user-images.githubusercontent.com/16653571/37007431-4aba6384-2103-11e8-9cd8-c89209710236.png)
angular.min.js:118 Error: [$interpolate:interr] http://errors.angularjs.org/1.5.8/$interpolate/interr?p0=%3C%5Bsrc%5D%3E&p1=Error%3A%20%5B%24sce%3Ainsecurl%5D%20http%3A%2F%2Ferrors.angularjs.org%2F1.5.8%2F%24sce%2Finsecurl%3Fp0%3Dhttps%253A%252F%252Fstorage.googleapis.com%252Foppiatestserver-resources%252FhDbEbaai2heT%252Fassets%252Faudio%252Fcontent-en-06m2cwerll6t.mp3

--- a/core/templates/dev/head/components/forms/AudioPlayerDirective.js
+++ b/core/templates/dev/head/components/forms/AudioPlayerDirective.js
@@ -17,21 +17,21 @@
  */
 
 oppia.directive('audioPlayer', [
-    'UrlInterpolationService', function(UrlInterpolationService) {
-      return {
-        restrict: 'E',
-        scope: {
-          src: '@',
-        },
-        templateUrl: UrlInterpolationService.getDirectiveTemplateUrl(
-          '/components/forms/audio_player_directive.html'),
-        controller: ['$scope', '$element', function($scope, $element) {
-          var audio = $element.find('audio')[0];
-          $scope.$on('$locationChangeStart', function(event) {
-            // The audio is paused when a change in route is detected.
-            audio.pause();
-          });
-        }]
-      };
-    }
+  'UrlInterpolationService', function(UrlInterpolationService) {
+    return {
+      restrict: 'E',
+      scope: {
+        getSourceUrl: '&sourceUrl',
+      },
+      templateUrl: UrlInterpolationService.getDirectiveTemplateUrl(
+        '/components/forms/audio_player_directive.html'),
+      controller: ['$scope', '$element', function($scope, $element) {
+        var audio = $element.find('audio')[0];
+        $scope.$on('$locationChangeStart', function(event) {
+          // The audio is paused when a change in route is detected.
+          audio.pause();
+        });
+      }]
+    };
+  }
 ]);

--- a/core/templates/dev/head/components/forms/audio_player_directive.html
+++ b/core/templates/dev/head/components/forms/audio_player_directive.html
@@ -1,1 +1,1 @@
-<audio controls preload="metadata" controlsList="nodownload" ng-src="<[src]>"></audio>
+<audio controls preload="metadata" controlsList="nodownload" ng-src="<[getSourceUrl()]>"></audio>

--- a/core/templates/dev/head/components/forms/audio_translations_editor_directive.html
+++ b/core/templates/dev/head/components/forms/audio_translations_editor_directive.html
@@ -14,7 +14,7 @@
           <[getAudioLanguageDescription(languageCode)]>
         </div>
         <div class="col-lg-8 col-md-8 col-sm-8">
-          <audio-player src="<[getAudioTranslationFullUrl(audioTranslation.filename)]>">
+          <audio-player source-url="getAudioTranslationFullUrl(audioTranslation.filename)">
           </audio-player>
         </div>
         <div class="col-lg-1 col-md-1 col-sm-1">

